### ISSUE TITLE
[Bug]: App log-in getting stuck in authentification on Google Pixel 7

### ISSUE THREAD

**Issue by HighTac** [astubenbord/paperless-mobile#336]
### What happened?

I'm running Paperless on a DS224+ and it has worked flawlessly for multiple months. It still does via browser, but the Android App has stopped working (on my Google Pixel 7). Whenever I try to log into my account, it gets stuck, the authentification circle is still circling, but that's it. I tried to reinstall, but the issue remains.

### Steps to reproduce

Go to Android App
Try to log in to Paperless Acc

### Log output from the app

```shell
2023-12-17 20:11:36.501	DEBUG   --- [      AuthenticationCubit] - login                    : Trying to log in Mox@19***03...
2023-12-17 20:11:36.502	DEBUG   --- [      AuthenticationCubit] - _addUser                 : Adding new user Mox@19***03..
2023-12-17 20:11:36.502	DEBUG   --- [      AuthenticationCubit] - _addUser                 : Fetching bearer token from the server...
```

### Paperless-ng*x Version

1.17.4

### Paperless-Mobile Version

3.1.8

### Device

Google Pixel 7

### Operating System Version

Android 14

### Device Locale

German

### Selected Locale

None

**Comment by astubenbord** [comment]
Seems like I forgot to add a timeout to the operation, I'll check.

**Comment by HighTac** [comment]
> Seems like I forgot to add a timeout to the operation, I'll check.

Thank you! If I understand it correctly that would not solve the issue for me, but I'd just get an error instead of being stuck in the authentification window?

**Comment by astubenbord** [comment]
Right, currently I have no way of knowing at which step the authentication process actually fails.

**Comment by armanckeser** [comment]
FWIW, I had the same problem, and figured out that my reverse proxy would only route to paperless if the request was from someone already authenticated within the reverse proxy. I removed the authentication requirement (from the proxy for paperless) and was able to login

**Comment by demdamdum** [comment]
> FWIW, I had the same problem, and figured out that my reverse proxy would only route to paperless if the request was from someone already authenticated within the reverse proxy. I removed the authentication requirement (from the proxy for paperless) and was able to login

could you please explain what exactly is to do? I have a synology and get the same error. I can login in via web browser but not the app.

**Comment by astubenbord** [comment]
Closing like #468 since this also does not seem to be directly related to the app but the setup. Feel free to open another issue if you still encounter this.
